autotst/reaction_test.py

```diff
@@ -152,6 +152,25 @@ def test_labeled_reaction(self):
         self.assertTrue(merged.get_labeled_atoms("*2")[0].is_hydrogen())
         self.assertTrue(merged.get_labeled_atoms("*1")[0].is_oxygen)
 
+    def test_labeled_reaction2(self):
+        """
+        Testing a reaction with the finicky [CH...] radical
+        """
+
```

Comment:
```suggestion
maybe a more detailed comment saying what this test is doing?  Because this is not just testing that you can sub your `[CH]` to `[CH...]` but that after that, if you can match the H abstraction reaction template and get the labeled reaction.  If you want to just test just that you can do the name change, you can probably not go through the `get_labeled_reaction()` to save time.
```